crates/nu-protocol/src/config/table.rs

```diff
@@ -383,6 +389,9 @@ impl Default for TableConfig {
             abbreviated_row_count: None,
             footer_inheritance: false,
             missing_value_symbol: "❎".into(),
+            batch_duration: Duration::from_secs(1),
+            // SAFETY: 1000 is non-zero.
+            stream_page_size: unsafe { NonZeroU16::new_unchecked(1000) },
```

Comment:
```suggestion
            stream_page_size: const { NonZeroU16::new(1000).expect("Non zero integer") },
```
Surprisingly `Option::expect()` is a const function!